client/src/components/BookingForm/style.css

```diff
@@ -0,0 +1,71 @@
+.booking {
+    background-image: linear-gradient(rgba(0, 0, 0,0.8),
+    rgba(0, 0, 0, 0.8)),url(../../assets/images/booking.svg);
+    background-repeat: no-repeat;
+    background-size:cover;
+    padding-top: 2rem;
+    }
+
+    .booking .ant-form {
+        margin-left: 20rem;
+        padding: 0px;
+    }
+    
+    .booking .first-name{
+        width: 12rem !important;
+        margin-right: 1rem
+    }
+
+    .booking .last-name{
+        width: 12rem !important;
+    }
+
+    .booking  h1.ant-typography{
+        color: #fff;
+    }
+
+.booking .ant-input::-webkit-input-placeholder{
+        color: #fff;
+    }
+
+.booking .ant-input{
+    color:#fff;
+    display: flex;
+    flex-wrap: wrap;
+    border-radius: 2rem;
+    background-color: rgba(0, 0, 0, 0.3);
+    -webkit-box-shadow: 0 0 0 2px rgba(87,168,233, .2);
+    box-shadow: 0 0 0 2px rgba(87,168,233, .2);
+    height: 3rem;
+    width: 25rem;
+}
+
+.booking .ant-btn{
+    background-color: #52CDDE;
+    color: #fff;
+    font-size: 2rem;
+    border-radius: 2rem;
+    width: 12rem;
+    height: 3.5rem;
+    margin: -40rem 0  0 -18rem;
+}
+
+.booking .err{
+    width: 25rem;
+    padding-bottom: 1rem;
+    margin-bottom: 1rem;
+}
+
+.input-height{
+    height: 5rem !important;
+}
+
+@media only screen and (max-width: 770px){
+    .booking .ant-form {
+        margin: 0 ;
+    }
+
+    .booking .ant-btn{
+        margin: 0;
+    }
+}
```

Comment:
We have a problem in our eslint configuration with CSS ,So please add a line here and pay attention to css file every time :(